packages/connectors-lib/src/power-apps.js

```diff
@@ -0,0 +1,114 @@
+import { ClientCredentials } from 'simple-oauth2'
+import Config from './config.js'
+import pkg from 'node-fetch'
+const fetch = pkg.default
+
+/*
+ * Access to dynamics using the OAuth2 client credentials flow.
+ * See https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow
+ */
+let accessToken
```

Comment:
Is `token` being here in the global scope intentional? Would this 'work' if we had multiple instances of the server sharing the load? If not, please consider using Redis/the DB so that all instances of the app can access the token.

Reply:
Each instance can grab a different, but valid token each with its own expiry time.